cannon/README.md

```diff
@@ -39,25 +39,24 @@ make cannon
 # Note:
 #  - The L2 RPC is an archive L2 node on OP goerli.
 #  - The L1 RPC is a non-archive RPC, also change `--l1.rpckind` to reflect the correct L1 RPC type.
-./bin/cannon run
---pprof.cpu
---info-at '%10000000'
---proof-at never
---input ./state.json
---
-../op-program/bin/op-program
---l2 http://127.0.0.1:8745
---l1 http://127.0.0.1:8645
---l1.trustrpc
---l1.rpckind debug_geth
---log.format terminal
---l2.head 0xedc79de4d616a9100fdd42192224580daee81ea3d6303de8089d48a6c1bf4816
---network goerli
---l1.head 0x204f815790ca3bb43526ad60ebcc64784ec809bdc3550e82b54a0172f981efab
---l2.claim 0x530658ab1b1b3ff4829731fc8d5955f0e6b8410db2cd65b572067ba58df1f2b9
---l2.blocknumber 8813570
---datadir /tmp/fpp-database
---server
+./bin/cannon run \
+  --pprof.cpu \
+  --info-at '%10000000' \
+  --proof-at never \
+  --input ./state.json \
+  -- ../op-program/bin/op-program \
```

Comment:
I'd probably undo this change and go back to having the `--` on a line by itself.  It's an important detail that is much easier to see when it's on its own line.

Reply:
Hmm bash requires a backslash at the end of the line to make it a valid multiline command. This makes a copy-paste of this not work in bash. I will revert, but much easier to have this to allow copy + pasting the command.

Reply:
you should keep the backslash, but retain `-- \` as a separate line.

Reply:
Ah got it, already removed but I will follow up with the change in a new pr